tequilapi/endpoints/identities.go

```diff
@@ -490,4 +597,7 @@ func AddRoutesForIdentities(
 	router.GET("/identities/:id/beneficiary", idmEnd.Beneficiary)
 	router.GET("/identities/:id/referral", idmEnd.GetReferralToken)
 	router.GET("/identities/:id/referral-available", idmEnd.ReferralTokenAvailable)
+
+	router.POST("/identities-export/:id", idmEnd.Export)
```

Comment:
I'm still not sure how we are going to protect it from being used by anyone.
Tequilapi opened with no auth for anything on the same host. Any script/tool/app can import and export private keys with no password.
I'd as a user prefer to have this secure than convenient for exporting by any app.